Makefile

```diff
@@ -16,7 +16,8 @@ all: kops
 
 .PHONY: channels examples
 
-DOCKER_REGISTRY?=gcr.io/must-override
+#DOCKER_REGISTRY?=gcr.io/must-override
+DOCKER_REGISTRY=prashima
```

Comment:
Do you plan to checkin with this docker_registry? Can this not be taken as a parameter?